roles/openshift_node/files/networkmanager/99-origin-dns.sh

```diff
@@ -44,8 +44,8 @@ if [[ $2 =~ ^(up|dhcp4-change|dhcp6-change)$ ]]; then
   # couldn't find an existing method to determine if the interface owns the
   # default route
   def_route=$(/sbin/ip route list match 0.0.0.0/0 | awk '{print $3 }')
-  def_route_int=$(/sbin/ip route get to ${def_route} | awk '{print $3}')
-  def_route_ip=$(/sbin/ip route get to ${def_route} | awk '{print $5}')
+  def_route_int=$(/sbin/ip route get to ${def_route} | awk -F 'dev' '{print $2}' | awk '{print $1}')
```

Comment:
this seems suitable, let's pipe the output of each ip command to head -n1 before awk to filter out cache line.

Reply:
@michaelgugino Thank you for your point. You're right. It include the new line, so head -n1 can remove it. I updated !